board/freescale/mx6ullevk/mx6ullevk.c

```diff
@@ -28,25 +27,43 @@
 
 DECLARE_GLOBAL_DATA_PTR;
 
-#define UART_PAD_CTRL  (PAD_CTL_PKE | PAD_CTL_PUE |		\
-	PAD_CTL_PUS_100K_UP | PAD_CTL_SPEED_MED |		\
-	PAD_CTL_DSE_40ohm   | PAD_CTL_SRE_FAST  | PAD_CTL_HYS)
+#define SKL_BOARD_TYPE_PCB195 0
+#define SKL_BOARD_TYPE_PCB213 1
+#define SKL_BOARD_TYPE_PCB217 2
+#define SKL_BOARD_TYPE_PCB277 3
```

Comment:
These macros duplicate the `SKL_BOARD_TYPE_E` enum from `board/freescale/mx6ullevk/siklu_api.h`, and removed in a subsequent commit.

Reply:
So what needs to be changed now?

Reply:
Modify the commit the needlessly introduced these macros to clean up the history, and tidy affected commits.